src/components/sorting-selector.yml

```diff
@@ -2,6 +2,9 @@ sortingSelector:
   changeSortingValue:
     en: Change sorting
     de: Sortierung ändern
-  changeSortingDirection:
-    en: Change sorting direction
-    de: Sortierungsrichtung ändern
+  changeSortingDirectionAscending:
+    en: Change to ascending order
+    de: In aufsteigende Reihenfolge sortieren
```

Comment:
For the German version, I think a similar "Change to" phrasing might make it clearer for the user that it is not the current sorting direction, but the one that will be applied. Please help 😊

Reply:
I agree. The best I can think of would be: "Zu (auf|ab)steigender Sortierung wechseln"

Reply:
Changed, thanks you!